habitat/plan.sh

```diff
@@ -41,6 +41,8 @@ do_before() {
 }
 
 do_download() {
+  build_line "Setting up the safe directory for the build"
+  git config --global --add safe.directory /src
```

Comment:
I'm experiencing a need for this in Windows as well, but not sure if your git path edit is fixing it.